src/docs/development/data-and-backend/state-mgmt/simple.md

```diff
@@ -420,7 +420,7 @@ rebuild when `notifyListeners` is called.
 ## Putting it all together
 
 You can [check out the
-example]({{site.github}}/filiph/samples/tree/provider-shopper/provider_shopper)
+example]({{site.github}}flutter/samples/tree/master/provider_shopper)
```

Comment:
There should be a slash before "flutter". But I'm glad to have this in a more official repo!

Reply:
Thanks for noticing! Fixed.